AmplifyPlugins/Analytics/HostApp/Configuration/amplifyconfiguration.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
- please delete this (move to trash) to remove the reference from HostApp as well, and then `amplify remove` or `amplify delete` your amplify project. this file should not be exposed to the public. the reason why `.gitignore` did not catch this is probably because we have the rule `*amplifyconfiguration.json` that catches all files that end in `amplifyconfiguration.json` And has a prefix. 
- can you add `amplifyconfiguration.json` and `awsconfiguration.json` to the top level .gitignore file?

Reply:
Added